services/system/Transact/src/RTransact.cpp

```diff
@@ -622,6 +623,18 @@ std::optional<AccountNumber> RTransact::getUser(HttpRequest request)
    return {};
 }
 
+struct Query
```

Comment:
Is transact the right place to put this? Maybe `explore` or even `common` (`/common/tapos/head` is kind of related)

Reply:
Hah, I had it in explore and moved it here because I thought it was general enough to make sense here. What do you think? `/common/chainid`?